src/util/retry.rs

```diff
@@ -7,6 +7,34 @@ use rand::Rng;
 
 /// A function that performs and retries the given operation, according to a retry policy and a set
 /// of retriable errors.
+///
+/// **Caution**: A retry policy without attempts capped by `MaxAttemptsRetryPolicy` decorator will
+/// result in infinite retries.
+///
+/// **Usage**:
+/// ```rust
+/// # use std::time::Duration;
+/// # use vss_client::error::VssError;
+/// # use vss_client::util::retry::{ExponentialBackoffRetryPolicy, retry, RetryPolicy};
+/// #
+/// # let base_delay = Duration::from_millis(10);
+/// # 		let max_attempts = 3;
+/// # 		let max_total_delay = Duration::from_secs(60);
+/// # 		let max_jitter = Duration::from_millis(5);
```

Comment:
I think it would be better to inline these since they are just repeated in the method names.

Reply:
I mean that the method and variable names are essentially the same. It would better to inline the values because the example would then demonstrate the types that the methods take.